### PR TITLE
Make it work

### PR DESCRIPTION
**Pull request by bruno-**
This PR adds the boilerplate to make this project work as a proper tmux plugin.

### CODE REVIEW

**Comment by erlandsona** [comment]
Cool so I got the whole make it a bash executable file with the shebang at the top. That said couple things I'm curious about...
Does it need to be a .sh extension or will the shebang be fine?

Also I just made a new commit that should make it more universally acceptable. That said... it forces the user to use the Meta key instead of something user prescribed? Is there a way to set the optional meta key to a user prescribed variable?

**Comment by bruno-** [comment]
> Does it need to be a .sh extension or will the shebang be fine?

It's fine with the shebang. Also we need the file to have `*.tmux` extension so it works as a tmux plugin compatible with [tpm](https://github.com/tmux-plugins/tpm).

**Comment by bruno-** [comment]
> it forces the user to use the Meta key instead of something user prescribed? Is there a way to set the optional meta key to a user prescribed variable?

A constraint to this is terminal. From what I know it only recognizes these character and char-combos:
1. ascii characters
2. ctrl + ascii characters
3. meta + ascii char
   4.. meta + ctrl + ascii char

First 2 groups above are already used by a lot of terminal applications (eg vim). That leaves you with meta char groups, and that is what this plugin is doing. Meta is usually `alt` character on the keyboard.

I think the only way to change this is if the user re-maps `alt` to some other key on the keyboard eg `F2`. Leaving `alt` as-is is probably the best though.

tl;dr; meta/alt key is a sensible way to go.
